src/internal/operators/groupBy.ts

```diff
@@ -71,16 +66,16 @@ export function groupBy<T, K, R>(keySelector: (value: T) => K, elementSelector?:
  * import { of } from 'rxjs';
  * import { mergeMap, groupBy, map, reduce } from 'rxjs/operators';
  *
- * of<Obj>(
+ * of(
  *   {id: 1, name: 'JavaScript'},
  *   {id: 2, name: 'Parcel'},
  *   {id: 2, name: 'webpack'},
- *   {id: 1, name: 'TypeScript'}
+ *   {id: 1, name: 'TypeScript'},
  *   {id: 3, name: 'TSLint'}
  * ).pipe(
  *   groupBy(p => p.id, p => p.name),
  *   mergeMap( (group$) => group$.pipe(reduce((acc, cur) => [...acc, cur], ["" + group$.key]))),
```

Comment:
Not really part of the PR but I noticed this extra space `mergeMap( (group$)...`. Can we get it removed?

And one more thing. Is there any reason why `group$.key` is send as a string and later parsed to an integer? If the string is required, does it make sense then to make it ``[`${group$.key}`]`` instead?

Reply:
Thanks for the feedback @dzhavat, made both of those changes :+1:

Reply:
Looks better 👍 

I'm actually not part of the RxJS team, so I commented more as a fellow contributor rather than someone who is responsible for reviewing the PR :)